src/net/sourceforge/kolmafia/request/ClosetRequest.java

```diff
@@ -1,5 +1,9 @@
 package net.sourceforge.kolmafia.request;
 
+import static net.sourceforge.kolmafia.request.ClosetRequest.ClosetRequestType.CLOSET_TO_INVENTORY;
+import static net.sourceforge.kolmafia.request.ClosetRequest.ClosetRequestType.INVENTORY_TO_CLOSET;
+import static net.sourceforge.kolmafia.request.ClosetRequest.ClosetRequestType.REFRESH;
+
```

Comment:
Are these because the previous code did not qualify the static class variable with ClosetRequest, so you couldn't qualify the enum constant with ClosetRequestType? I mean you could - and I think I'd prefer that to importing some (but not all) of tge enum constants.

Reply:
yes, they were simply to minimise changes in the refactor. I could qualify instead.